### PR TITLE
Initialize React web app 

### PR DESCRIPTION
**Pull request by arthurborgesdev**
# In this Pull Request I implemented

- Initialization of the project using create-react-app
- Clean of files that won't be used
- Render a simple App container with a "Hello, React!" h1 heading content

## To run the project

- Clone the repo at: https://github.com/arthurborgesdev/book-a-teacher-webapp
- Change branch to `feature/initialize-react-web-app`
- Run `yarn start` at the Terminal
- Open `http://localhost:3000` in a Browser
- The message `Hello, React!` should be displayed on the screen

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,8 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <h1>Hello, React!</h1>
```
Comment (fernandorpm) [inline src/App.js:5]: Starting fresh and clean! That is great :sunglasses:

File: src/index.js [hunk 2]
```diff
@@ -11,7 +9,3 @@ ReactDOM.render(
   document.getElementById('root')
 );
 
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
```
Comment (fernandorpm) [inline src/index.js:17]: Nice job again removing the unnecessary lines here as well.

**Review by fernandorpm — APPROVED** [review]
# :heavy_check_mark: Pull Request Approved! :heavy_check_mark:
WOW WHAT IS THIS PERFECTION??????? WERE HUMANS SUPPOSED TO BEHOLD SUCH POWER????

Congratulations on your hard work! Please remember to merge the project and don't forget that you're awesome! :heart:




Cheers and Happy Coding! :heart:

File: src/App.js (hunk 1 above)
Comment (arthurborgesdev) [reply src/App.js:5]: Thank you! :)

**Pull request merged**